scripts/src/server.sh

```diff
@@ -130,6 +132,17 @@ if [ -z "$CONTAINER_NAME" ]; then
   CONTAINER_NAME="${CONTAINER_NAME/:/-}-ssh"
 fi
 
+if [[ ! -f "${SSH_KEY_FILE}" ]]; then
+  if [[ "${SSH_KEY_FILE}" != "${DEFAULT_SSH_RSA}" ]]; then
+    echo "Provided SSH key file ${SSH_KEY_FILE} does not exist."
+    exit 1
+  fi
+  if [[ ! -f "${DEFAULT_SSH_ED25519}" ]]; then
+    echo "SSH not set up! Configure SSH on your system."
+    exit 1
+  fi
+  SSH_KEY_FILE="${DEFAULT_SSH_RSA}"
```

Comment:
```suggestion
  else
    SSH_KEY_FILE="${DEFAULT_SSH_ED25519}"
  fi
```

Reply:
^ with the else statement I think it's a bit more readable

Reply:
do you prefer a `if ... else if ... else` then? I am not a huge fan of `else` in case of `exit` conditions but the `if ... else` here would only make sense with the `else if` in the middle I would say.